policy/framework.go

```diff
@@ -548,11 +548,33 @@ func ResolveFramework(mrn string, frameworks map[string]*Framework) *ResolvedFra
 		Nodes:         map[string]ResolvedFrameworkNode{},
 	}
 
+	// note: this code will disable a control if it is disabled in any framework in the hirearchy
+	// (asset or space-level). This should be fine for now since we don't have a use-case of enabling
+	// an already disabled control on a lower level in the hierarchy.
+	disabledControls := map[string]struct{}{}
+	for _, framework := range frameworks {
+		for _, group := range framework.Groups {
+			// If the group is not an exception group or has been rejected, skip over it
+			if (group.Type != GroupType_DISABLE && group.Type != GroupType_OUT_OF_SCOPE) ||
```

Comment:
We have similar code in the policy resolver: https://github.com/mondoohq/cnspec/blob/main/policy/resolver.go#L1504-L1523

I'm not sure whether that's actually needed here.

Reply:
I am also not entirely sure when that other code is executed. I know for sure though that this bit here determines whether we report scores for controls